core/modules/parsers/wikiparser/rules/codeblock.js

```diff
@@ -1,4 +1,5 @@
-/*\
+// @ts-check
```

Comment:
What does this do?

Reply:
It enables VSCode and ts cli regard this file as `.ts` file. Maybe this enable different parse mode of VSCode.

Reply:
The [docs](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html#ts-check) seem to imply that one can also use a separate "jsconfig.json" file. Would that be possible here?

Reply:
I try again and find VScode works with or without this. What make it work is open both file as tab...

I need to experiment more.